ct.yaml

```diff
@@ -0,0 +1,11 @@
+# See https://github.com/helm/chart-testing-action
+# See https://github.com/helm/chart-testing#configuration
+remote: origin
+target-branch: main
+chart-dirs:
+  - charts
+chart-repos: []
+helm-extra-args: --timeout 800s
+check-version-increment: false
+validate-maintainers: false
+validate-yaml: true
```

Comment:
IMO all the checks should be enabled. Any other opinions?

Reply:
I agree.